app/main/posts/detail/post-detail-data.directive.js

```diff
@@ -111,10 +113,17 @@ function PostDetailDataController(
                     .sortBy('priority')
                     .value();
 
-                angular.forEach(attributes, function (attr) {
-                    this[attr.key] = attr;
+                attributes.forEach(attr => {
```

Comment:
ah so I forgot to explain why this happened in the first place. 
We had attributes in an object with their id as the key, so they were not sorted, so what I did was convert that to a sorted array of objects (attributes is sorted a few lines before this) and change the view to use that correctly